webpack.config.js

```diff
@@ -11,10 +11,7 @@ module.exports = {
     port: 5000,
   },
   output: {
-    library:{
-      name: "greenstand",
-      type: "var"
-    },
+    library: { name: 'greenstand', type: 'umd' },
```

Comment:
yea, I always get confused by the import/export stuff, like umd, comm...

Reply:
I don't totally understand the differences either, I just kept looking at examples and trying different things. I can say at least that changing `umd` back to `var` will break it. I'm not sure if `umd` is the best choice but it's probably fine.